includes/privacy.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+namespace WordCamp\Budgets\Privacy;
+
+use WP_Query;
+use WordCamp\Budgets\Reimbursement_Requests;
+
+defined( 'WPINC' ) || die();
+
+
+add_filter( 'wp_privacy_personal_data_exporters', __NAMESPACE__ . '\register_personal_data_exporters' );
+add_filter( 'wp_privacy_personal_data_erasers', __NAMESPACE__ . '\register_personal_data_erasers' );
+
+/**
+ * Registers the personal data eraser for each WordCamp post type
+ * @param array $erasers
+ *
+ * @return array
+ */
+function register_personal_data_erasers( $erasers ) {
+	/**
```

Comment:
I probably wouldn't say in the code comment that we will never add an eraser for this, because we will be adding anonymization for non-business critical data. Perhaps just note that this is a placeholder.